app/views/dashboard/index.html.erb

```diff
@@ -140,7 +140,7 @@
               </div>
             </div>
             <div class="box-body text-center float-center">
-              <h3 class="text-center"><%= @recent_donations.by_source(:diaper_drive).count %> Diaper Drives <%= display_interval %> for a total of <%= number_with_delimiter(@recent_donations.by_source(:diaper_drive).sum { |d| d.line_items.total }) %> items collected</h3>
+              <h3 class="text-center"><%= @recent_donations.by_source(:diaper_drive).map(&:diaper_drive_participant).uniq.size %> Diaper Drives <%= display_interval %> for a total of <%= total_received_from_diaper_drives %> items collected</h3>
```

Comment:
Do you want to call `.uniq` on this? If they had 20 diaper drives and 15 of those were at one location won't it show as 6 diaper drives in the period not 20?

Reply:
I think this is one of the areas that we need to revisit with Diaper Drives. The way we have it, it could be entered in a way that treats Diaper Drive Participants as either:
 - the diaper drive itself, assuming diaper drives are separate efforts (so all donations coming in from diaper drive X get associated with Diaper Drive Participant X)
 - an organization that contributes to a unnamed diaper drive (Diaper Drive Participants X, Y and Z all contribute to a general "diaper drive")

it's possible that *I* just don't have a correct understanding of how it works, too :) I think the UI should provide a little more guidance about the expectation of what the things mean

Oh -- I do agree that the language is incorrect, though -- it says "diaper drives" but really it's tracking "number of diaper drive participants" -- I'll fix that now so at least the language is consistent.

I don't know how to track how many individual diaper drives there are (assuming they can be separate things and are not just an ongoing effort).

Reply:
👍